app/src/main/java/to/bitkit/ui/screens/wallets/receive/ReceiveLiquidityScreen.kt

```diff
@@ -57,7 +97,9 @@ fun ReceiveLiquidityScreen(
         Spacer(Modifier.height(24.dp))
 
         Column(
-            modifier = Modifier.padding(horizontal = 16.dp)
+            modifier = Modifier
+                .padding(horizontal = 16.dp)
+                .verticalScroll(rememberScrollState())
```

Comment:
Need `.weight(1f)` here too, otherwise we get this bug on large phone screens:


| Before | After |
| - | - |
| <img width="312" src="https://github.com/user-attachments/assets/b42d9e2a-50b7-4f8f-9471-4d13c3511764" /> | <img width="312" src="https://github.com/user-attachments/assets/35a58c2a-bed0-4c0e-a315-9e80d7fbf5ac" /> |

Reply:
Pushed the fixes